api/app/signals/apps/api/tests/test_private_category_endpoint.py

```diff
@@ -297,3 +299,98 @@ def test_patch_category_multiple_unchanged_slo_calls(self):
 
         category.refresh_from_db()
         self.assertEqual(category.slo.count(), slo_count_at_start_of_tests + 1)
+
+    def test_patch_category_public_parameters(self):
+        """
+        Change the public_name and/or the public_accessible parameters of a Category
+        """
+        self.client.force_authenticate(user=self.sia_read_write_user)
+
+        url = f'/signals/v1/private/categories/{self.parent_category.pk}'
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self._assert_category_data(category=self.parent_category, data=response.json())
+
+        response = self.client.patch(url, data={'public_name': 'Public name', 'public_accessible': True})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
```

Comment:
Super nitpicky, but ...

If the update fails, the assertions in `self._assert_category_data` below will not fail. Add a check that the response (or the category instance) contains the changed data.

comment applies to the other PATCHes as well

Reply:
If the update fails the status code will not be 200. It is a 400. So this will not be an issue.
If this really is an issue I want to update this in another PR because then it is a general issue and not only applicable to this PR.

Reply:
I agree that most likely you will get a 400 on failure - hence my comment about nitpicky, but the point I was making was:

The test will still work if you replace ```response = self.client.patch(url, data={'public_name': 'Public name', 'is_public_accessible': True})``` with a no-op.